src/generator-vulcanjs/generators/start/index.js

```diff
@@ -0,0 +1,10 @@
+const chalk = require('chalk');
+const VulcanGenerator = require('../../lib/VulcanGenerator');
+
+module.exports = class extends VulcanGenerator {
+
+  default() {
+    this.log(chalk.green('\nStarting your app... \n'));
+    this.spawnCommandSync('meteor', ['run']);
```

Comment:
My only concern was about using the various assert / errors or not. 

I did not use them, so you're not forced to use the scaffolding tool, `start` just does the job.